opm/grid/cpgrid/CpGrid.cpp

```diff
@@ -3060,11 +3059,11 @@ void CpGrid::populateRefinedCells(std::vector<Dune::cpgrid::EntityVariableBase<c
             for (int corn = 0; corn < 8; ++corn) {
                 int refinedCorn;
                 const auto& preAdaptCorn = preAdapt_cell_to_point[corn];
-                if ( elemLgrAndElemLgrCorner_to_refinedLevelAndRefinedCorner.count({elemLgr, preAdaptCorn}) == 0 ) {
+                if ( elemLgrAndElemLgrCorner_to_refinedLevelAndRefinedCorner.find({elemLgr, preAdaptCorn}) == elemLgrAndElemLgrCorner_to_refinedLevelAndRefinedCorner.end() ) {
                     // Corner might have vanished - Search its equivalent lgr-corner in that case -
                     // last lgr where the corner appears -
                     std::array<int,2> lastAppearanceLgr_lgrCorner = {0, 0}; // It'll be rewritten
-                    if(markedElemAndEquivRefinedCorn_to_corner.count({elemLgr, preAdaptCorn}) == 1) {
+                    if(markedElemAndEquivRefinedCorn_to_corner.find({elemLgr, preAdaptCorn}) != markedElemAndEquivRefinedCorn_to_corner.end()) {
```

Comment:
Please store iterator returned by find and use that to get the value. see other comment.